pages/events/_event.vue

```diff
@@ -9,6 +9,11 @@
           {{currentEvent.location}}
         </p>
         <div class="mt-3 mb-5">
+          <v-btn
+            v-if="currentEvent.registrationUrl"
```

Comment:
```suggestion
            v-if="currentEvent.registrationUrl && !currentEvent.registration"
```
We don't want to show both internal and external registrations. Also this requires an additional field in Storyblok to manage

Reply:
Also, when creating a PR, please let maintainers to commit changes, it'd be faster if I could apply the suggestion myself and merge the PR